doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -28,8 +28,8 @@ The following table illustrates ABI compatibility between different versions of
 
    * - |NCS| versions
      - Compatible nRF54H20 SoC binaries version
-   * - |NCS| v2.9.0-nRF54H20
-     - `nRF54H20 SoC Binaries v0.8.0`_, compatible with the nRF54H20 DK v0.9.0 and later revisions.
+   * - |NCS| v2.9.0-nRF54H20-1
+     - `nRF54H20 SoC Binaries v0.9.0`_, compatible with the nRF54H20 DK v0.9.0 and later revisions.
```

Comment:
If you wanna add this here, you have to also add the corresponding link to the bundle in the links.txt file. If you search for nRF54H20 SoC Binaries v0. you will find the block where all the soc binaries links are located.